### PR TITLE
Add import script

### PR DESCRIPTION
**Pull request by kovrus**

### CODE REVIEW

File: github-archive/README.md [hunk 1]
```diff
@@ -27,3 +27,18 @@ data provided with Crate. Requires ```npm``` to work.
...
+
+```sh
+  export AWS_ACCESS_KEY_ID='...'
+  export AWS_SECRET_ACCESS_KEY='...'
+```
+
+Import data from s3 to the Crate cluster:
+
+```sh
+python import.py --start 2011/8 --end 2011/10 --host localhost:4200
```
Comment (chaudum) [inline github-archive/README.md]: this is not the buildout way to invoke scripts.
please add a console script entry point in `setup.py`
see: https://github.com/crate/crash/blob/master/setup.py#L71

File: github-archive/import.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python
+
+import argparse
+import sys, os
```
Comment (chaudum) [inline github-archive/import.py]: don't import multiple modules with a single import, split them up in

``` py
import os
import sys
```

File: github-archive/import.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import datetime
```
Comment (chaudum) [inline github-archive/import.py]: remove module import

File: github-archive/import.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+  parser.add_argument('-s','--start', help='start date', required=True)
+  parser.add_argument('-e','--end', help='end date', required=True)
+  parser.add_argument('-host','--host', help='host', required=True)
+  args = parser.parse_args()
+ 
+  start_arg = args.start.split('/')
+  end_arg = args.end.split('/')
+
+  start_date = datetime.date(int(start_arg[0]), int(start_arg[1]), 1)
```
Comment (chaudum) [inline github-archive/import.py]: use `date(...)`

File: github-archive/import.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+import datetime
+from datetime import date, timedelta
+from crate import client
+from urllib import quote_plus
+
+def delta_month(date, months):
+    return date + datetime.timedelta(months*365/12)
+
+def main():
+  parser = argparse.ArgumentParser()
```
Comment (chaudum) [inline github-archive/import.py]: extract argument parsing into separate method

File: github-archive/import.py [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  aws_secret_key = os.environ['AWS_SECRET_ACCESS_KEY']
+  aws_access_key = os.environ['AWS_ACCESS_KEY_ID']
+  
+  connection = client.connect(args.host)
+  cur = connection.cursor()
+
+  for single_date in (start_date + timedelta(n) for n in range((delta_month(end_date, 1) - start_date).days)):
+    import_data = single_date.strftime("%Y-%m-%d");
+    month_partition = single_date.strftime("%Y-%m");
+    query = """COPY github PARTITION (month_partition="{0}")
```
Comment (chaudum) [inline github-archive/import.py]: use sql parameter substitution instead of string formatting!

File: github-archive/import.py [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+  start_date = datetime.date(int(start_arg[0]), int(start_arg[1]), 1)
+  end_date = datetime.date(int(end_arg[0]), int(end_arg[1]), 1)
+
+  aws_secret_key = os.environ['AWS_SECRET_ACCESS_KEY']
+  aws_access_key = os.environ['AWS_ACCESS_KEY_ID']
+  
+  connection = client.connect(args.host)
+  cur = connection.cursor()
+
+  for single_date in (start_date + timedelta(n) for n in range((delta_month(end_date, 1) - start_date).days)):
```
Comment (chaudum) [inline github-archive/import.py]: extract the query generation into separate method so you can unittest it

**Comment by chaudum** [comment]
ack

**Comment by mfussenegger** [comment]
indentation seems to be mixed. Please stick to 4 whitespaces for indentation

File: github-archive/s3tocrate.py [hunk 8]
```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
+
+import argparse
+import os
+import sys
+from datetime import date, timedelta, datetime
+from crate import client
+from urllib import quote_plus
```
Comment (mfussenegger) [inline github-archive/s3tocrate.py:8]: this will fail in python3

**Pull request merged**